php/src/php/Config.h

```diff
@@ -120,6 +123,10 @@ ZEND_END_MODULE_GLOBALS(ice)
 ZEND_BEGIN_ARG_INFO(ice_void_arginfo, 0)
 ZEND_END_ARG_INFO()
 
+// An arginfo used for __toString() methods.
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO(ice_to_string_arginfo, IS_STRING, 0)
+ZEND_END_ARG_INFO()
+
```

Comment:
This is breaking PHP 7.x build, I would keep this 8.2 only too
```suggestion
// An arginfo used for __toString() methods.
#if PHP_VERSION_ID >= 80200
 ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO(ice_to_string_arginfo, IS_STRING, 0)
 ZEND_END_ARG_INFO()
+#else
// The return type declaration is optional with PHP < 8.2
ZEND_BEGIN_ARG_INFO(ice_to_string_arginfo, 0)
ZEND_END_ARG_INFO()
#endif
```